packages/@react-aria/overlays/src/usePreventScroll.ts

```diff
@@ -13,19 +13,33 @@
 import {useEffect} from 'react';
 
 /**
- * Prevents scrolling on the document body on mount, and 
+ * Prevents scrolling on the document body on mount, and
  * restores it on unmount. Also ensures that content does not
  * shift due to the scrollbars disappearing.
  */
-export function usePreventScroll(): void {
+
+interface PreventScrollOptions {
+  isOpen?: boolean
+}
+
+export function usePreventScroll(options? : PreventScrollOptions) {
+
+  const {isOpen = false} = options;
```

Comment:
```suggestion
  const {isOpen = true} = options;
```
IMO this should be flipped to true so that pre-existing components will continue to work if they call usePreventScroll without any arguments.

Reply:
that sounds fine but isn't that essentially what the above change is doing?

Reply:
Ah, I see the difference now (thanks @snowystinger). IMO it feels a bit weird that we would have two different defaults dependent on whether or not options are provided or not. Is there a use case you had in mind?

Reply:
Just backward compatibility really...

Reply:
Hello! I'm going to work on this tonight. I'm really sorry for the delay I've been away for days now. Thanks for your help I really appreciate.

Reply:
@McZenith No worries, thanks for working on this! You can go ahead and implement the format that @devongovett suggested above.